src/main/java/com/socialmdm/service/SocialMediaService.java

```diff
@@ -1,132 +1,201 @@
 package com.socialmdm.service;
 
 import java.io.IOException;
-import java.util.List;
 import java.util.UUID;
 
-import javax.ws.rs.CookieParam;
-import javax.ws.rs.FormParam;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
+import javax.servlet.http.HttpServletResponse;
+import javax.websocket.server.PathParam;
 
-import twitter4j.Query;
-import twitter4j.QueryResult;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+//import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import com.socialmdm.util.Constants;
+
+import twitter4j.FilterQuery;
+import twitter4j.StallWarning;
 import twitter4j.Status;
-import twitter4j.Twitter;
-import twitter4j.TwitterException;
-import twitter4j.TwitterFactory;
+import twitter4j.StatusDeletionNotice;
+import twitter4j.StatusListener;
+import twitter4j.TwitterStream;
+import twitter4j.TwitterStreamFactory;
 import twitter4j.conf.ConfigurationBuilder;
 
-import com.google.api.services.pubsub.Pubsub;
-import com.google.api.services.pubsub.model.PublishRequest;
-import com.google.api.services.pubsub.model.PublishResponse;
-import com.google.api.services.pubsub.model.PubsubMessage;
-import com.google.appengine.api.channel.ChannelMessage;
-import com.google.appengine.api.channel.ChannelService;
-import com.google.appengine.api.channel.ChannelServiceFactory;
-import com.google.appengine.repackaged.com.google.common.collect.ImmutableList;
-import com.socialmdm.util.PubsubUtils;
-
-@Path("/social")
+/**
+ * SocialMediaService for SocialMedia related operations
+ * 
+ * @author naveen
+ *
+ */
+@Controller
 public class SocialMediaService {
 
-    @GET
-    @Path("/getToken")
-    public String getToken() {
+    Logger logger = Logger.getLogger(SocialMediaService.class);
+
+    private SimpMessagingTemplate messagingTemplate;
+    private static PubSubService pubSubService = setPubSubService();
+    private static TwitterStream twitterStream = null;
```

Comment:
Why PubSubService and TwitterStream a static variable? If only one object need to be created then it should be singleton